app/models/user.js

```diff
@@ -34,6 +36,9 @@ userSchema.methods.asJson = function (options, cb) {
     json.createdAt = this.created_at
     json.updatedAt = this.updated_at
     json.flags = this.flags
+
+    json.isAdmin = this.role.includes(USER_ROLES.ADMIN)
+    json.isUser = this.role.includes(USER_ROLES.USER)
```

Comment:
Is `.isUser` planned to be used for anything specific? or can we assume that setting`.isAdmin` to true is the distinction between whether a user has elevated privileges or not?

Reply:
Oh, sure. I think my thought process was that a user could have multiple user roles hence the array of roles and therefore we needed to know if a user was admin and/or user. 
Never mind then. Do you think we still need the `USER_ROLES.USER` then?